### ISSUE TITLE
Bug: Currently active menu item is not conveyed programmatically in the header

### ISSUE THREAD

**Issue by andrewhick** [alphagov/tech-docs-gem#403]
When documentation includes links in the header, for example on the [WCAG Primer page](https://alphagov.github.io/wcag-primer/index.html), the currently active menu item, such as Overview or All Requirements, is conveyed visually (by changing the text from white to blue) but not programmatically.

While it meets [WCAG 1.4.1 Use of Colour](https://www.w3.org/WAI/WCAG22/Understanding/use-of-color) (because the contrast ratio between white and blue is at least 3:1) it fails [WCAG 1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships) because information is not conveyed programmatically - for example by using `aria-current`.

![Menu with most items in white and All Requirements in blue](https://github.com/user-attachments/assets/50c641bb-b97f-4925-ab00-ecde2450c5ac)

This was originally raised as [WCAG Primer issue 132](https://github.com/alphagov/wcag-primer/issues/132).
